liquidctl.8

```diff
@@ -195,6 +207,37 @@ Internal data used by some drivers.
 .
 .SH DEVICE SPECIFICS
 .
+.SS Corsair Commander Pro
+.SS Corsair Lighting Node Pro
+Cooling channels: \fIfan\fR, \fIfan[1\-5]\fR. (Commander Pro only)
+.PP
+Lighting channels: \fIled1\fR, \fIled2\fR.
+.TS
+l c
+---
+l c .
+Mode	#colors
+\fIclear\fR	0
+\fIoff\fR	0
+\fIfixed\fR	1
+\fIcolor_shift\fR	2
+\fIcolor_pulse\fR	2
+\fIcolor_wave\fR	2
+\fIvisor\fR	2
+\fIblink\fR	2
+\fImarquee\fR	1
+\fIsequential\fR	1
+\fIrainbow\fR	0
+\fIrainbow2\fR	0
+.TE
+.PP
+When applicable the animation speed can be set with
+.BI \-\-speed= value ,
+where the allowed values are: \fIfast\fR, \fImedium\fR, \fIslow\fR.
+.PP
+When applicable the animation direction can be set with
+.BI \-\-direction= value ,
+where the allowed values are: \fIforward\fR or \fIbackward\fR.
```

Comment:
`--temperature-sensor`, `--start-led` and `--maximum-leds` should also be mentioned in these notes.

You can also omit the second (or later) "When applicable".